alpenhorn/auto_import.py

```diff
@@ -470,6 +533,22 @@ def _import_file(node, root, acq_name, file_name):
                     "empty. Leaving fields NULL." % (acq_name)
                 )
                 di.CorrAcqInfo.create(acq=acq)
+    if atype == "hfb" and ftype.name == "hfb":
+        if not acq.corrinfos.count():
+            try:
+                di.CorrAcqInfo.create(
+                    acq=acq, **get_acqcorrinfo_keywords_from_h5(fullpath)
+                )
+                log.info(
+                    'Added information for correlator acquisition "%s" to '
```

Comment:
```suggestion
                    'Added information for HFB acquisition "%s" to '
```
"Correlator" refers to the `corr` file type in this case